flink-contrib/flink-connector-wikiedits/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
+			<artifactId>flink-clients_${scala.binary.version}</artifactId>
```

Comment:
Why does this module need a dependency on `flink-clients`? Is it being executed?